media-sound/jack-smf-utils/jack-smf-utils-1.0-r2.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="Utilities for MIDI streams and files using Jack MIDI"
+HOMEPAGE="https://sourceforge.net/projects/jack-smf-utils/"
+SRC_URI="https://sourceforge.net/projects/jack-smf-utils/files/${PN}/${PV}/${P}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
```

Comment:
Also, if tests passed (if they even exist, add them if they exist upstream btw), I'm happy to take your word for it and add ~arm in a new commit

Reply:
Noted. Sorry if this next part is long, it's mostly FTR and pasted it to save me doing it over and over.

Built with FEATURES=test if that is the correct procedure. Package does not appear to have tests.

jack-smf-player and jack-smf-recorder were tested manually with few different midi files and with a running jack (media-sound/jack2) session to fluidsynth and they appeared to play and record midi.

Here are some example commands;

# start the jack server daemon
jackd -r 48000 -d alsa -d hw:1 -r 44100 -X alsa_seq -p 512

# as a reference, test the midi file with fluidsynth on its own.
fluidsynth -r 48000 -l -j -a jack -m jack /usr/share/sounds/sf2/FluidR3_GM.sf2 Stayin_Alive.mid
# to test the player/recorder, start a fluidsynth instance with jack audio/midi and a soundfont
fluidsynth -r 48000 -l -j -a jack -m jack /usr/share/sounds/sf2/FluidR3_GM.sf2

# start a recording instance, this will wait for midi input on the jack port
# jack-smf-recorder:midi_in and then send any events to the file specified
jack-smf-recorder test-recording.mid

# start up a player and connect to fluidsynth or the recorder, then wait for the midi file
# to finish, the recorded midi file should play the same as the original file..

# to hear through fluidsynth played with the jack-smf-player
jack-smf-player -s -a fluidsynth-midi:midi_00 Stayin_Alive.mid

# playback a file into the recorder to test recording
jack-smf-player -s -a jack-smf-recorder:midi_in Stayin_Alive.mid

# pkill/ctrl+c the jack-smf-recorder if necessary, then playback the recorded midi file through
# fluidsynth to check if anything sounds funny
jack-smf-player -s -a fluidsynth-midi:midi_00 test-recording.mid


Also to note, there is a fork of jack-smf-utils with newer commits/features on github, I did look at that as possible new upstream but I believe that fork still includes a bundled libsmf.
I don't see a reason why there should be a bundled lib?

Pushing new branch shortly,  I appreciate the feedback, many thanks.